skyvern/forge/sdk/services/bitwarden.py

```diff
@@ -157,8 +157,8 @@ async def _get_secret_value_from_url(
             BitwardenService.sync()
             session_key = BitwardenService.unlock(master_password)
 
-            # Extract the domain from the URL and search for items in Bitwarden with that domain
-            domain = tldextract.extract(url).domain
+            # Extract the domain(with suffix) from the URL and search for items in Bitwarden with that domain
+            domain = tldextract.extract(url).registered_domain
```

Comment:
can we fall back to the domain if the registered_domain search doesn't return anything?